eng/pipelines/maintenance-packages.yml

```diff
@@ -55,83 +55,41 @@ extends:
-          parameters:
-            artifacts:
-              publish:
-                artifacts: false
-                logs: true
-                manifests: true
-            enableMicrobuild: true
-            enablePublishUsingPipelines: true
-            publishAssetsImmediately: true
-            enablePublishTestResults: true
-            testResultsFormat: vstest
-            enableSourceIndex: ${{ and(eq(variables._RunAsInternal, True), eq(variables['Build.SourceBranch'], 'refs/heads/main')) }}
-            workspace:
-              clean: all
-            jobs:
-            - job: build_and_test_windows
-              displayName: 'Build and test Windows'
-              timeoutInMinutes: 60
-              pool:
-                name: $(DncEngInternalBuildPool)
-                image: $(WindowsImage)
-                os: windows
-              preSteps:
-              - checkout: self
-                fetchDepth: 0
-                clean: true
-              steps:
-              - script: eng\common\cibuild.cmd
-                  -configuration Release
-                  -prepareMachine
-                  $(_AdditionalBuildArgs)
-                  $(_InternalBuildArgs)
-                displayName: $(_BuildJobDisplayName)
-
-      # Official rolling build (internal/main) - does not run tests
-
-      - ${{ if and(eq(variables._RunAsInternal, True), eq(variables['Build.SourceBranch'], 'refs/heads/internal/main')) }}:
-        - template: /eng/common/templates-official/jobs/jobs.yml@self
-          parameters:
-            artifacts:
-              publish:
-                artifacts: false
-                logs: true
-                manifests: true
-            enableMicrobuild: true
-            enablePublishUsingPipelines: true
-            publishAssetsImmediately: true
-            enablePublishTestResults: true
-            testResultsFormat: vstest
-            enableSourceIndex: ${{ and(eq(variables._RunAsInternal, True), eq(variables['Build.SourceBranch'], 'refs/heads/internal/main')) }}
-            workspace:
-              clean: all
-            jobs:
-            - job: build_and_pack_windows
-              displayName: 'Build and pack Windows'
-              timeoutInMinutes: 60
-              pool:
-                name: $(DncEngInternalBuildPool)
-                image: $(WindowsImage)
-                os: windows
-              preSteps:
-              - checkout: self
-                fetchDepth: 0
-                clean: true
-              steps:
-              - script: eng\common\cibuild.cmd
-                  -configuration Release
-                  -prepareMachine
-                  /p:SkipTests=true
-                  $(_AdditionalBuildArgs)
-                  $(_InternalBuildArgs)
-                displayName: $(_BuildJobDisplayName)
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
+        parameters:
+          artifacts:
+            publish:
+              artifacts: false
```

Comment:
I wonder if this needs to change.  We can see once we start producing packages if those are published or not.

Reply:
I'd like to see the results after merging.